assets/performanceprofile/scripts/set-ns-rps-mask.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+MASK=${1}
+REMOUNT="false"
+
+if ! [[ -w /sys ]]; then
+  if ! res=$(mount -o remount,rw /sys 2>&1); then
+    logger "${0}: Failed to remount /sys as rw: ${res}"; exit 0;
+  fi
+  REMOUNT="true"
+fi
+
+# /sys/class/net can't be used recursively to find the rps_cpus file, use /sys/devices/virtual instead
+if ! res=$(find /sys/devices/virtual -type f -name rps_cpus -exec sh -c "echo ${MASK} > {}" \; 2>&1) || [[ -n $res ]]; then
```

Comment:
for dev in /sys/devices/virtual/net/*; do
          echo ${mask} > ${dev}/queues/rx-0/rps_cpus || logger "${0}: Failed to apply the RPS mask on ${dev}:"
done